jepsen/scalardb/test/scalardb/scalardb_test.clj

```diff
@@ -8,34 +8,53 @@
             [scalardb.transfer :as transfer]
             [jepsen [core :as jepsen]]))
 
+(def nodes ["n1" "n2" "n3" "n4" "n5"])
```

Comment:
Does it only work with 5-node setting ? (or this clock-drift only work with 5-node setting ?)
Is there any way to make it work with arbitrary number of nodes ?

Reply:
Got it. Thank you !